nickspaargaren/racetijden-ios
# Add CI with SwiftLint

### What
- Adds CI with [SwiftLint](https://github.com/realm/SwiftLint).
- Fixes code style

### How to test
- Create a not so pretty change anywhere in the code
- Commit and push
- See that the pipeline fails
- Run `swiftlint --fix` and lint the code.
- Commit and push
- See that the pipeline passes
